docs/source/documentation.rst

```diff
@@ -181,6 +113,19 @@ Below is a short tutorial describing how to use the AxonDeepSeg plugin for Napar
 
    <iframe width="700" height="394" src="https://www.youtube.com/embed/zibDbpko6ko" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+
+Python Shell use
+-----------------------------------------
+
+To use AxonDeepSeg in a Python shell, first activate your virtual environment::
+
+    ads_activate
+
+.. NOTE :: To switch back to your default environment, run:
+  ::
+
+       conda deactivate
+
```

Comment:
I'm wondering how useful this section is on its own. Since we expose the `axondeepseg_*` scripts already, is there a reason for users to activate the conda venv besides updating ADS?

(I might be missing some additional usage of ADS here...)

Reply:
I guess it's more for devs, specifically internally. Our wiki is pretty out of date - I'll update the title of this section to make it more explicit that its for developers